db/db_test2.cc

```diff
@@ -2388,6 +2409,7 @@ TEST_F(DBTest2, TestPerfContextIterCpuTime) {
   // CPU timing is not enabled with kEnableTimeExceptForMutex
   SetPerfLevel(PerfLevel::kEnableTimeExceptForMutex);
   Iterator* iter = db_->NewIterator(ReadOptions());
+  ASSERT_OK(iter->status());
```

Comment:
Move to after the last iter->Valid() check

Reply:
Done.